includes/class-convertkit-output-restrict-content.php

```diff
@@ -0,0 +1,709 @@
+<?php
+/**
+ * ConvertKit Output Restrict Content class.
+ *
+ * @package ConvertKit
+ * @author ConvertKit
+ */
+
+/**
+ * Restricts (or displays) a single Page, Post or Custom Post Type's content
+ * based on the Post's "Restrict Content" configuration.
+ *
+ * @since   2.1.0
+ */
+class ConvertKit_Output_Restrict_Content {
+
+	/**
+	 * Holds the success message to display on screen as a notification.
+	 *
+	 * @since   2.1.0
+	 *
+	 * @var     bool|string
+	 */
+	private $success = false; // @phpstan-ignore-line.
```

Comment:
Why does this need an ignore line?

Reply:
Not ignoring it results in a false positive, as the property is read in the `views` folder, which doesn't have static analysis performed on:
![Screenshot 2023-01-06 at 14 02 44](https://user-images.githubusercontent.com/1462305/211027693-bd83eb62-bdf6-4783-863e-46ab1668300a.png)

Performing static analysis on the views folder results in numerous false positives, due to PHPStan not being able to (presently) detect which views are loaded within which classes
![Screenshot 2023-01-06 at 14 06 16](https://user-images.githubusercontent.com/1462305/211028063-f36338cd-e097-4bef-997d-4c897bd5dd83.png)